frontend/src/components/ThreeEngine/Particle.js

```diff
@@ -0,0 +1,96 @@
+import * as THREE from "three";
+
+export class Particle {
```

Comment:
Can we eventually convert this file to TS to make it strongly typed or is that a nono?

Reply:
yeah def should be ts